test/model/workloads/WT-13252.workload

```diff
@@ -0,0 +1,20 @@
+create_table(1, table, Q, Q)
```

Comment:
@pmacko86, I just want to double-check, does the RTS model run as expected on 8.0?

Reply:
The version in 8.0 does not support the `evict` operation, so we shouldn't include this workload file. (That is, unless there are plans to backport it, but that would surprise me.)

Reply:
Good to know, thank you!